client/reducer/user.js

```diff
@@ -0,0 +1,46 @@
+import axios from 'axios';
+import { browserHistory } from 'react-router';
+
+const GET_USER = 'GET_USER';
+const REMOVE_USER = 'REMOVE_USER';
+
+const defaultUser = {};
+
+const getUser = user => ({ type: GET_USER, user });
+const removeUser = () => ({ type: REMOVE_USER });
+
+export const me = () =>
+  dispatch =>
+    axios.get('/auth/me')
+      .then(res =>
+        dispatch(getUser(res.data || defaultUser)));
+
+export const auth = (email, password, method) =>
+  dispatch =>
+    axios.post(`/auth/${method}`, { email, password })
```

Comment:
I find it **extremely** suspect that you have a _verb_ in your _resource_ path. What is `method` supposed to be?